anchor/src/state.rs

```diff
@@ -0,0 +1,32 @@
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use lido::util::serialize_b58;
+use serde::Serialize;
+use solana_program::{account_info::AccountInfo, entrypoint::ProgramResult, pubkey::Pubkey};
+
+#[repr(C)]
+#[derive(
+    Clone, Debug, Default, BorshDeserialize, BorshSerialize, BorshSchema, Eq, PartialEq, Serialize,
+)]
+pub struct Anchor {
+    /// The SPL Token mint address for bSOL.
```

Comment:
We should add a version byte, like we do with Lido.

Oh and about the version byte ... if we deserialize the Lido instance in this program, we need to check the version byte, and bail out if it doesn’t match (which means we upgraded the Lido program but not yet this program).